config/features/glance.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  glance:
+    flags: ""
+    create_kvm_image:
```

Comment:
Wouldn't it be better to call this `kvm_test_image`?  By de-coupling the name from the scenario, it will allow reusing of the image for other scenarios.